### PR TITLE
Fix clone workspace

### PR DESCRIPTION
**Pull request by duianto**
### problem
Trying to clone a workspace `SPC l w c`
Does create a new workspace,
but it just shows the scratch buffer.

### notes
In addition to fixing the cloning.

Show a minibuffer message:
`Workspace: 1, cloned to Workspace: 2`

To make it clear that the cloning worked.
Because the only visible difference, is that
one of the left side mode line numbers changed.

### CODE REVIEW

File: layers/+spacemacs/spacemacs-layouts/funcs.el [hunk 1]
```diff
@@ -701,6 +701,20 @@ STATE is a window-state object as returned by `window-state-get'."
...
+  "Clone the current workspace.
+And show a minibuffer message, ex:
+Workspace: 1, cloned to Workspace: 2"
+  (interactive)
+  (let* ((eyebrowse-new-workspace nil) ; nil = clone current workspace
+         (current-workspace-nr (eyebrowse--get 'current-slot))
+         (window-configs (eyebrowse--get 'window-configs))
+         (slots (mapcar 'car window-configs))
+         (next-free-workspace-nr (eyebrowse-free-slot slots)))
+    (eyebrowse-create-window-config)
```
Comment (lebensterben) [inline layers/+spacemacs/spacemacs-layouts/funcs.el:714]: By calling `eleybrowse-create-window-config`, you are repeating L711-713.
Why not to directly call `(eyebrowse-switch-to-window-config next-free-workspace-nr)` instead?

```elisp
(defun eyebrowse-create-window-config ()
  "Creates a window config at a yet unoccupied slot."
  (interactive)
  (let* ((window-configs (eyebrowse--get 'window-configs))
         (slots (mapcar 'car window-configs))
         (slot (eyebrowse-free-slot slots)))
    (eyebrowse-switch-to-window-config slot)))
```

Comment (duianto) [reply layers/+spacemacs/spacemacs-layouts/funcs.el]: Fixed. Your right, thanks.

**Review by lebensterben — APPROVED** [review]

**Review by syl20bnr — APPROVED** [review]

**Comment by syl20bnr** [comment]
Tested in my config.

**Comment by syl20bnr** [comment]
Thank you ! 🚀 
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request merged**